scripts/scil_plot_meanstdperpoint.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Plot mean/std per point.
+"""
+
+import argparse
+import json
+import os
+
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_output_dirs_exist_and_empty)
+from scilpy.utils.metrics_tools import plot_metrics_stats
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('input_json',
+                   help='JSON file containing the mean/std per point. For '
+                        'example, can be created using '
+                        'scil_compute_metrics_along_streamline.')
+    p.add_argument('output_dir',
+                   help='Output directory')
+
+    p.add_argument('--fill_color',
+                   help='Hexadecimal RGB color filling the region between '
+                        'mean ± std. The hexadecimal RGB color should be '
+                        'formatted as 0xRRGGBB.')
+    p.add_argument('--nb_points', type=int, default=20,
+                   help='Number of points defining the centroid streamline.')
+
+    add_overwrite_arg(p)
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, [args.input_json])
+    assert_output_dirs_exist_and_empty(parser, args, args.output_dir,
+                                       create_dir=True)
+
+    if args.fill_color and len(args.fill_color) != 8:
+        parser.error('Hexadecimal RGB color should be formatted as 0xRRGGBB')
+
+    with open(args.input_json, 'r+') as f:
+        meanstdperpoint = json.load(f)
+
+    for bundle_name, bundle_stats in meanstdperpoint.items():
+        for metric, metric_stats in bundle_stats.items():
+            num_digits_labels = len(str(args.nb_points))
+            means = []
+            stds = []
+            for label_int in range(1, args.nb_points+1):
```

Comment:
From what I can see, the nbr of points is indicated in the json. This information is known, that's why I was saying the parameter was useless

Reply:
agree